ui/job-view/details/tabs/TabsPanel.jsx

```diff
@@ -167,6 +169,7 @@ class TabsPanel extends React.Component {
               ) : (
                 <Tab disabled>Test Groups</Tab>
               )}
+              <Tab>Full Description</Tab>
```

Comment:
nit: to save UI space, can we make this `Description` ?

Reply:
We have an 'Artifacts and Debugging' tab and this is about debugging. The artifacts list can get long but putting the instructions how to run the task locally above the artifacts if preferable. (If the instructions are below, the might be missed. The same applies to the artifacts, maybe there should be a first line which tells users to scroll down for artifacts?)

Reply:
what's existing "Debugging Tools" ?
if there's no "debugging" right now, could we make the existing tab "Artifacts", and add "Debugging" as separate tab?